internal/gapicgen/execv/command.go

```diff
@@ -42,3 +44,22 @@ func (c *CmdWrapper) Run() error {
 	log.Printf("[%s] >>>> %v <<<<", c.Dir, strings.Join(c.Args, " ")) // NOTE: we have some multi-line commands, make it clear where the command starts and ends
 	return c.Cmd.Run()
 }
+
+// ForEachMod runs the given function with the directory of
+// every non-internal module.
+func ForEachMod(rootDir string, fn func(dir string) error) error {
```

Comment:
Does this need to be moved? Doesn't really fit with the `execv` package given it doesn't run commands. Doesn't appear to be used anywhere other than the file it was defined in?

Reply:
It does not have to for this PR, but there is some logic in genbot that could be refactored to use it in a future PR around updating dependencies. It was not a super great fit I agree, but it is also always used in the context of running commands so is seemed like an okay fit as a helper method for commands? I can move it back and just have some copy/paste if you think that would be better.

Reply:
SGTM.